units.libs.lua

```diff
@@ -183,7 +183,7 @@ StaticLibrary {
 		Filters = {
 			{ Pattern = "[/\\]windows[/\\]"; Config = { "win32-*", "win64-*" } },
 			{ Pattern = "[/\\]macosx[/\\]"; Config = "mac*-*" },
-			{ Pattern = "[/\\]x11[/\\]"; Config = { "x11-*" } },
+			{ Pattern = "[/\\]X11[/\\]"; Config = { "x11-*" } },
```

Comment:
Ah of course. OK, you want to fix it for Windows if you haven't already, and I'll rename it on linux next time I try to build it?